src/main/java/pl/noip/lolstats/lol/stats/controller/RegistrationController.java

```diff
@@ -0,0 +1,23 @@
+package pl.noip.lolstats.lol.stats.controller;
+
+import org.springframework.web.bind.annotation.*;
+import pl.noip.lolstats.lol.stats.dto.RegistrationRequest;
+import pl.noip.lolstats.lol.stats.dto.Repository;
+
+@RestController
+@RequestMapping("/register")
+public class RegistrationController {
+
+    private Repository repository;
+
+    public RegistrationController(Repository repository) {
+        this.repository = repository;
+    }
+
+    @PostMapping
+    public void register(@RequestBody RegistrationRequest registrationReguest) {
+        System.out.println(registrationReguest.getEmail());
+        System.out.println(registrationReguest.getPassword());
+        repository.save(registrationReguest);
```

Comment:
check if email exists in database

Reply:
check is email is in correct form

Reply:
check if password has at least 4 characters

Reply:
Password sha256 hash should be saved in database. You shouldn't use same class for data transport, and database model